js/src/chrome/users/chrome.users.js

```diff
@@ -489,6 +499,7 @@ Wu.Chrome.Users = Wu.Chrome.extend({
 
 	_createInviteInput : function (options) {
 
+		var me = this;
```

Comment:
@IgorZiegler  please don't use `me`... `this` is perfectly fine, and it's what's being used everywhere else in the code. It's confusing to have `me` as context. thanks.